web/tests/lib/zjquery_element.js

```diff
@@ -62,6 +62,10 @@ function FakeElement(selector, opts) {
                 selector,
             };
         },
+        each(cb) {
+            cb.call($self);
+            return $self;
+        },
```

Comment:
This doesn't look like a loop at all? Maybe I'm missing something.

I don't think you need tests specifically for this block of code -- manual testing should be sufficient -- but let's try not to add a test helper function that doesn't behave as described.

Reply:
I think the function `update_presence_indicators` should be stubbed out.